project_scrum/models/project_task.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+    _order = "sequence"
+
+    @api.model
+    def _get_moscow_field(self):
+        return [('1', 'must'),
+                ('2', 'should'),
+                ('3', 'could'),
+                ('4', 'wont'),
+                ('not_set', 'Not Set')]
+
+    @api.model
+    def _get_value_field(self):
+        return [
+            ('0', '0'),
+            ('1', '1'),
+            ('2', '2'),
+            ('3', '3'),
+            ('5', '5'),
+            ('8', '8'),
+            ('13', '13'),
+            ('20', '20'),
+            ('40', '40'),
+            ('100', '100'),
+            ('00', 'Not Set'),
+        ]
+
+    @api.model
+    def _get_risk_field(self):
+        return [
+            ('0', '0'),
+            ('1', '1'),
+            ('2', '2'),
+            ('3', '3'),
+            ('5', '5'),
+            ('8', '8'),
+            ('13', '13'),
+            ('20', '20'),
+            ('40', '40'),
+            ('100', '100'),
+            ('00', 'Not Set'),
+        ]
+
+    @api.model
+    def _get_kano_field(self):
+        return [
+            ('excitement', 'Excitement'),
+            ('indifferent', 'Indifferent'),
+            ('mandatory', 'Mandatory'),
+            ('performance', 'Performance'),
+            ('questionable', 'Questionable'),
+            ('reverse', 'Reverse'),
+            ('not_set', 'Not Set'),
+        ]
+
+    actor_ids = fields.Many2many(
+        comodel_name='project.scrum.actors',
+        string='Actor',
+    )
+    sprint_id = fields.Many2one(
+        comodel_name='project.scrum.sprint',
+        string='Sprint',
+    )
+    us_id = fields.Many2one(
+        comodel_name='project.scrum.us',
+        string='User Stories',
+        index=True,
+    )
+    use_scrum = fields.Boolean(related='project_id.use_scrum')
+    current_sprint = fields.Boolean(
+        compute='_compute_current_sprint',
+        string='Current Sprint',
+        search='_search_current_sprint',
+    )
+    moscow = fields.Selection('_get_moscow_field')
+    value = fields.Selection('_get_value_field')
+    risk = fields.Selection('_get_risk_field')
+    kano = fields.Selection('_get_kano_field')
+    color = fields.Integer(related='project_id.color')
+    reference = fields.Char(
```

Comment:
is this just some arbitrary code for tasks? If so, replace this by a dependency to https://github.com/OCA/project/tree/10.0/project_task_code